module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -15,23 +15,30 @@ Read the Base `R` documentation [**here**](https://www.rdocumentation.org/packag
+| Base `R`          | [`sum(x)`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/sum)                            | Sum               | Returns the sum of all values (values must be integer or numeric) in object `x` or logical vector `x`.                                                           |
+| Base `R`          | [`log(x)`](https://www.rdocumentation.org/packages/SparkR/versions/2.1.2/topics/log)                            | Logarithm.        | Gives the natural logarithm of object `x`. `log2(x)` can be used to give the logarithm of the object in base 2.                                                           |
 | Base `R`          | [`head()`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Head                  | Returns the top 6 rows of an object in the environment by default. You can specify how many rows you want by including the `n = `argument.                                                        |
 | Base `R`          | [`tail()`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Tail                  | Returns the bottom 6 rows of an object in the environment by default. You can specify how many rows you want by including the `n =` argument.                                                                  |
-| Base `R`          | [`as.factor(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/factor)       | As Factor             | Coerces object x into a factor (which is used to represent categorical data). This function can be used to coerce object `x` into other data types, i.e., `as.character`, `as.data.frame`, `as.matrix`, etc. |
-| Base `R`          | [`dim(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/dim)                | Dimensions summary    | Returns the dimensions of object `x`.                                                                                                                                                               |
+| Base `R`          | [`factor(x)` or `as.factor(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/factor)       | Factor             | Coerces object x into a factor (which is used to represent categorical data). This function can be used to coerce object `x` into other data types, i.e., `as.character`, `as.numeric`, `as.data.frame`, `as.matrix`, etc. |
+| Base `R`          | [`summary(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/summary)                | Object summary    | Returns a summary of the values in object `x`.                                                                                                                                                               |
```

Comment:
May want to put a note here or with `summarise` that explicitly warns users to not get confused between the two. I've noticed people do confuse these